install.sh

```diff
@@ -80,7 +80,13 @@ check() {
 
 case $SHELL in
 	*zsh) check ".zshrc" ;;
-	*bash) check ".bashrc" ;;
+	*bash) 
+		if [ -f "$HOME/.bashrc" ];
+		then
+			check ".bashrc"
+		else
+			check ".bash_profile"
+		fi;;
```

Comment:
```suggestion
	*bash) 
		[ -f "$HOME/.bashrc" ] && check $HOME/.bashrc
		[ -f "$HOME/.bash_profile" ] && check $HOME/.bash_profile
	;;
```

Reply:
Blindly accepted the change on GitHub, which broke functionality, but already fixed in [591d0ec](https://github.com/spicetify/spicetify-cli/pull/1583/commits/591d0ece9493b79fa9f342c14c73f2c6bd82a07c).

Reply:
oh crap, i forgot that `check` actually already does the `$HOME` path...